src/cloudai/_core/base_runner.py

```diff
@@ -288,17 +288,22 @@ async def monitor_jobs(self) -> int:
                 if self.mode == "dry-run":
                     successful_jobs_count += 1
                 else:
-                    job_status_result = self.get_job_status(job)
-                    if job_status_result.is_successful:
+                    if job.test.job_status_check:
```

Comment:
What do you think about creating a derivative of `BaseRunner` and overriding `monitor_jobs` method? Test will need a way to specify which Runner to use, currently it is assumed from the System.

Reply:
We can do that, but I don't think it is a solution to the problem that I want to solve. This is the issue that I want to solve: https://nvidia.slack.com/archives/C06LC3PRHDZ/p1720706394214339. The problem is that when we check the job status of all tests upon completion, if a job is killed due to dependency, it does not have valid output. As a result, we need a way to ignore the job status of some tests.

Reply:
If we can delay it till Monday, I suggest we have a call to discuss it. If urgent — go with version.

Reply:
I can wait. Let's discuss.